fpdf/image_parsing.py

```diff
@@ -311,6 +314,19 @@ def _decode_base64_image(base64Image):
 ]
 
 
+def iccp_is_valid(iccp):
```

Comment:
Could you please add this unit test to cover that case?
```python
def test_insert_jpg_with_invalid_icc(caplog):
    pdf = fpdf.FPDF()
    pdf.add_page()
    pdf.image(HERE / "jpg_with_invalid_icc.jpg")
    assert "ICC Profile" in caplog.text
    assert "is invalid" in caplog.text
```

And here is the code to generate `jpg_with_invalid_icc.jpg`:
```python
from PIL import Image, ImageCms
img = Image.open(HERE / "insert_images_insert_jpg_icc.jpg")
iccp = img.info["icc_profile"]
img.save(HERE / "jpg_with_invalid_icc.jpg", icc_profile=iccp[5:])
```

Reply:
thanks! done